napari/layers/vectors/_tests/test_vectors.py

```diff
@@ -121,6 +121,7 @@ def test_random_3D_vectors_image():
     assert layer._view_data.shape[2] == 2
 
 
+@pytest.mark.filterwarnings("ignore:Passing `np.nan`:DeprecationWarning:numpy")
```

Comment:
Is this not fixable?

Reply:
probably could/should be addressed in #1536 
the issue is that with empty vectors, `dims.range` gives `[(nan, nan, 1.0), (nan, nan, 1.0), (nan, nan, 1.0)]`... so these lines in dims:
https://github.com/napari/napari/blob/2315443d9bd4fb6b8cc30967cf433d017f7c8ca2/napari/components/dims.py#L270-L275

cause this warning
```
DeprecationWarning: Passing `np.nan` to mean no clipping in np.clip has always been unreliable, and is now deprecated. In future, this will always return nan, like it already does when min or max are arrays that contain nan. To skip a bound, pass either None or an np.inf of an appropriate sign.
```

Reply:
this is changing in #1532 too